gluten-flink/planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecCalc.java

```diff
@@ -14,137 +14,135 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.apache.flink.table.planner.plan.nodes.exec.stream;
 
+import org.apache.gluten.rexnode.RexNodeConverter;
 import org.apache.gluten.rexnode.Utils;
 import org.apache.gluten.table.runtime.operators.GlutenSingleInputOperator;
 import org.apache.gluten.util.LogicalTypeConverter;
-import org.apache.gluten.rexnode.RexNodeConverter;
-
-import io.github.zhztheplayer.velox4j.expression.TypedExpr;
-import io.github.zhztheplayer.velox4j.plan.FilterNode;
-import io.github.zhztheplayer.velox4j.plan.PlanNode;
-import io.github.zhztheplayer.velox4j.plan.ProjectNode;
+import org.apache.gluten.util.PlanNodeIdGenerator;
 
-import org.apache.calcite.rex.RexNode;
-import org.apache.flink.FlinkVersion;
-import org.apache.flink.api.dag.Transformation;
-import org.apache.flink.configuration.ReadableConfig;
 import org.apache.flink.table.data.RowData;
 import org.apache.flink.table.planner.delegation.PlannerBase;
-import org.apache.flink.table.planner.plan.nodes.exec.ExecNode;
-import org.apache.flink.table.planner.plan.nodes.exec.ExecNodeMetadata;
 import org.apache.flink.table.planner.plan.nodes.exec.ExecEdge;
+import org.apache.flink.table.planner.plan.nodes.exec.ExecNode;
 import org.apache.flink.table.planner.plan.nodes.exec.ExecNodeConfig;
 import org.apache.flink.table.planner.plan.nodes.exec.ExecNodeContext;
+import org.apache.flink.table.planner.plan.nodes.exec.ExecNodeMetadata;
 import org.apache.flink.table.planner.plan.nodes.exec.InputProperty;
 import org.apache.flink.table.planner.plan.nodes.exec.common.CommonExecCalc;
 import org.apache.flink.table.planner.plan.nodes.exec.utils.ExecNodeUtil;
 import org.apache.flink.table.planner.plan.nodes.exec.utils.TransformationMetadata;
 import org.apache.flink.table.runtime.operators.TableStreamOperator;
 import org.apache.flink.table.runtime.typeutils.InternalTypeInfo;
 import org.apache.flink.table.types.logical.RowType;
-import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
-import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
-import org.apache.gluten.util.PlanNodeIdGenerator;
 
 import javax.annotation.Nullable;
 
+import org.apache.calcite.rex.RexNode;
```

Comment:
Why org.apache.flink behind javax? I think the import order should like this:
org.apache.gluten.*

io.github
org.apache.calcite
org.apache.flink.
org.apache.flink.shaded

javax

java.uitl

Reply:
done

Reply:
Maybe you could finish this

Reply:
> Why org.apache.flink behind javax? I think the import order should like this: org.apache.gluten.*
> 
> org.apache.flink. org.apache.flink.shaded .* javax java You can refer to flink's code style, only add gluten to the first. This will keep flink's import order, so will not change files copied from flink too much.

copy from
https://github.com/apache/incubator-gluten/pull/9576#discussion_r2086341404

Reply:
done